src/databricks/labs/ucx/queries/migration/main/05_1_failed_table_migration.sql

```diff
@@ -0,0 +1,6 @@
+/* --title 'List of migration failures' --type table --width 6 */
+SELECT
+  message
+FROM inventory.logs
+WHERE
+  message LIKE 'Failed to migrate %'
```

Comment:
Rewrite the log message to have a similar structure as in your other PR: `<code>: <message>`